src/patch.js

```diff
@@ -22,6 +22,33 @@ var walker = require('./walker'),
     getWalker = walker.getWalker,
     setWalker = walker.setWalker;
 
+/**
+ * @const {boolean}
+ */
+var IS_PRODUCTION = process.env.NODE_ENV === 'production';
+
+if (!IS_PRODUCTION) {
+  var assertNoUnclosedTags = function(root) {
+    var openElement = getWalker().getCurrentParent();
+    if (!openElement) {
+      return;
+    }
+
+    var openElements = [];
+    while(openElement && openElement !== root) {
+      openElements.push(openElement);
```

Comment:
I think that is a fair point. I'd like to separate the concern of getting the ancestry from getting the tag names, but not a good enough reason to prevent IE8 usage.